plugins/course-apps/proctoring/Settings.jsx

```diff
@@ -84,12 +85,9 @@ const ProctoringSettings = ({ onClose }) => {
     } else if (name === 'proctoringProvider') {
       const newFormValues = { ...formValues, proctoringProvider: value };
 
-      if (value === 'proctortrack') {
+      if (allowedProctoringEscalationEmail.includes(value)) {
         setFormValues({ ...newFormValues, createZendeskTickets: false });
```

Comment:
```suggestion
        setFormValues({ ...newFormValues });
```
Since the zendesk integration is deprecated, it's safe to just remove createZendeskTickets from here.

Reply:
createZendeskTickets is removed now.